File: app/src/main/java/com/example/sl_terms/ResultActivity.java
package com.example.sl_terms;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.view.View;
import android.widget.TextView;

import static com.example.sl_terms.BusinessLogicTest.blt;

public class ResultActivity extends AppCompatActivity {




    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_result);


        String numberCorrectAnswersS = getIntent().getStringExtra("numberCorrectAnswersS");
        String countQuestionsS =" ";
        countQuestionsS = getIntent().getStringExtra("countQuestions");


     //   int id_student = Integer.parseInt(id_studentS);


        TextView textView = (TextView) findViewById(R.id.textView6);

     //   String result = Integer.toString(blt.numberOfCorrectAnswers(id_student));
        textView.setText(numberCorrectAnswersS +"/" +countQuestionsS);
    }
    @Override
    public void onBackPressed() {
        Intent intent = new Intent(ResultActivity.this, MainActivity.class);
        startActivity(intent);
        finish(); // закрываем эту активити
    }
}

File: app/src/main/java/com/example/sl_terms/ResultActivity.kt
package com.example.sl_terms

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.TextView
import com.example.sl_terms.ResultActivity

class ResultActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_result)
        val numberCorrectAnswersS = intent.getStringExtra("numberCorrectAnswersS")
        var countQuestionsS = " "
        countQuestionsS = intent.getStringExtra("countQuestions")
        //   int id_student = Integer.parseInt(id_studentS);
        val textView = findViewById<View>(R.id.textView6) as TextView
        //   String result = Integer.toString(blt.numberOfCorrectAnswers(id_student));
        textView.text = "$numberCorrectAnswersS/$countQuestionsS"
    }

    override fun onBackPressed() {
        val intent = Intent(this@ResultActivity, MainActivity::class.java)
        startActivity(intent)
        finish() // закрываем эту активити
    }
}